README.md

```diff
@@ -151,30 +151,42 @@ Example:
 curl hh3.gbdev.io/api/all?order_by=title&sort=desc
 ```
 
-## Deploy
+## Local Development
 
-### Requirements
+There are two options for running this project locally: manually or using Docker.
 
-You need Python 3 and a couple of packages to build psycopg2:
+### Manual requirements
 
+You need Python 3 and a couple of packages to build `psycopg2` (database driver).
+
+On Linux, this command should install all requirements:
 ```bash
 apt install python3 libpq-dev python3-dev python3-venv
 ```
 
-### Run server
+Next, install Postgres 12 ([download link](https://www.postgresql.org/download/)), create a user, password and an `hh` table. Have it running in the background on port `5432`.
```

Comment:
Are you sure the creation of the `hh` table is needed? Isn't that done by the migrations created by django?

Reply:
Oops, that is true, good catch! It's a typo, I meant to say "an hh **database**", not a table. The database is not created by default and is specified in the database URL at the end of the string: postgres://yourpostgresuserhere:yourpostgrespasswordhere@localhost:5432/**hh**